<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Monopoly</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnStart">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupbox1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string># of Players</string>
    </property>
    <widget class="QRadioButton" name="rb2players">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>2 Players</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb3players">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>3 Players</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb4players">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>4 Players</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbBoard">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>681</width>
      <height>571</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnRoll">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>440</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Roll</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTrade">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>440</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Trade</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>80</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>William: $2000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Alex: $2000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnProperties">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>440</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Manage Properties</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEnd">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>440</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>End Turn</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>490</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblDie1">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>330</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblDie2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>330</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblBoard">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>731</width>
       <height>431</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>board1.png</pixmap>
     </property>
    </widget>
    <zorder>lblBoard</zorder>
    <zorder>btnTrade</zorder>
    <zorder>btnRoll</zorder>
    <zorder>label_3</zorder>
    <zorder>label_2</zorder>
    <zorder>btnProperties</zorder>
    <zorder>btnEnd</zorder>
    <zorder>label_4</zorder>
    <zorder>lblDie1</zorder>
    <zorder>lblDie2</zorder>
   </widget>
   <widget class="QLabel" name="lblProperty">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>112</y>
      <width>341</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>gbBoard</zorder>
   <zorder>btnStart</zorder>
   <zorder>groupbox1</zorder>
   <zorder>lblProperty</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
